filo/src/containers/Home/home.module.scss

```diff
@@ -0,0 +1,4 @@
+* > body {
```

Comment:
Weird ass selector

Reply:
Just erased it, turns out we didn't need it after all